lib/appRelationsDiscovery.js

```diff
@@ -11,7 +11,7 @@ const defaultOptions = {
 /**
  * Validates and analyzes a list of AsyncAPI documents and get applications described by those files
  * 
- * @param {Array} asyncApiDocs An array of asyncApiDocuments
+ * @param {Array} asyncApiDocs An array of strings
```

Comment:
```suggestion
 * @param {String[]} asyncApiDocs An array of stringified AsyncAPI documents
```

this is what I meant, I'm not 100% sure on the jsdoc syntax, you will have to check how it generates into API.md

Reply:
Updated manually.